lib/cli/Cargo.toml

```diff
@@ -149,14 +149,17 @@ experimental-io-devices = [
 ]
 singlepass = [
     "wasmer-compiler-singlepass",
+    "wasmer/singlepass",
     "compiler",
 ]
 cranelift = [
     "wasmer-compiler-cranelift",
+    "wasmer/cranelift",
     "compiler",
 ]
 llvm = [
     "wasmer-compiler-llvm",
+    "wasmer/llvm",
```

Comment:
This is not correct

Reply:
Please undo

Reply:
Undone in https://github.com/wasmerio/wasmer/pull/3483/commits/8d3e7c0dadc638cec97265415a46c32f035ae073